src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyRepository.cs

```diff
@@ -21,6 +21,29 @@ public class MethodologyRepository : IMethodologyRepository
             _methodologyParentRepository = methodologyParentRepository;
         }
 
+        public async Task<Methodology> CreateMethodologyForPublication(Guid publicationId)
+        {
+            var methodology = (await _contentDbContext.Methodologies.AddAsync(new Methodology
+            {
+                Slug = publicationId.ToString(),
```

Comment:
Is this temporary until we get the slug from publication title or alternative title? Also as discussed I think we should move slug to MethodologyParent and migrate the existing slugs but could do that as a separate ticket / PR. This would benefit EES-2375 which needs to look up by slug.

Reply:
Yes, temporary :D  I intend to migrate to MethodologyParent as part of my next ticket.